diff --git a/base/BUILD.gn b/base/BUILD.gn
index bc18a7ef68c23614b3bbdd9968139e63a8e81f44..293478af1af4fea7068234f2f8d990f1f38274e9 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4838,6 +4838,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5499,6 +5500,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
