diff --git a/chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/FeedSurfaceMediator.java b/chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/FeedSurfaceMediator.java
index 90d3071a1fe92957b15ef73c756905e32d0827d5..aac7eb5e3df48c2e5321fba655c9081222d4a1e5 100644
--- a/chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/FeedSurfaceMediator.java
+++ b/chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/FeedSurfaceMediator.java
@@ -275,7 +275,7 @@ public class FeedSurfaceMediator
         // Start is showing, update the value of Pref.ENABLE_SNIPPETS_BY_DSE here. The
         // value should be updated before adding an observer to prevent an extra call of
         // updateContent().
-        getPrefService()
+        if (FeedFeatures.isFeedEnabled(mProfile)) getPrefService()
                 .setBoolean(
                         Pref.ENABLE_SNIPPETS_BY_DSE,
                         !mIsNewTabSearchEngineUrlAndroidEnabled
