diff --git a/chrome/browser/ui/views/page_action/page_action_properties_provider.cc b/chrome/browser/ui/views/page_action/page_action_properties_provider.cc
index 84f5e2000178e545c4ffe77f56e8d0b9ad457aa8..ccd3e57beb91b5d2ca3f1a459a7fc0dd0295d1b4 100644
--- a/chrome/browser/ui/views/page_action/page_action_properties_provider.cc
+++ b/chrome/browser/ui/views/page_action/page_action_properties_provider.cc
@@ -187,6 +187,13 @@ constexpr auto kPageActionProperties =
                 .type = PageActionIconType::kAutofillAddress,
             },
         },
+        {
+            kActionShowPartitionedStorage,
+            {
+                .histogram_name = "PartitionedStorage",
+                .type = brave::kPartitionedStorageActionIconType,
+            },
+        },
         {
             kActionVirtualCardEnroll,
             {
