diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
index 32fc35ca9080489e008938e950df597e9e622964..a258d916736367eed36ccf3c1449897cdb1d8c9e 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
@@ -1809,7 +1809,7 @@ public class ToolbarManager
         return mOmniboxFocusStateSupplier.get()
                 || (mToolbarPositionController != null
                         && mToolbarPositionController.doesPrefMismatchPosition())
-                || (getNewTabPageForCurrentTab() != null);
+                ;
     }
 
     private void back(int metaState, int buttonState) {
