diff --git a/chrome/browser/ui/webui/theme_source.cc b/chrome/browser/ui/webui/theme_source.cc
index cd2b07922ba29457558add16f8363152778c0927..7fb2fd70a85c8267fcda47923bef6654bd2da418 100644
--- a/chrome/browser/ui/webui/theme_source.cc
+++ b/chrome/browser/ui/webui/theme_source.cc
@@ -353,6 +353,7 @@ void ThemeSource::SendColorsCss(
 
   const auto* theme_service =
       ThemeServiceFactory::GetForProfile(profile_->GetOriginalProfile());
+  BRAVE_THEME_SOURCE_SEND_COLORS_CSS
 
   std::optional<std::string> css_content =
       GenerateColorsCss(color_provider, url, theme_service->GetIsGrayscale(),
