diff --git a/chrome/browser/ui/views/page_action/page_action_view.cc b/chrome/browser/ui/views/page_action/page_action_view.cc
index 1e803a8362d7151ccb0b1c55baa3a0f9337e7bf5..a90e54890d0e182d1591caaef8394e8f7ee586c8 100644
--- a/chrome/browser/ui/views/page_action/page_action_view.cc
+++ b/chrome/browser/ui/views/page_action/page_action_view.cc
@@ -135,7 +135,7 @@ void PageActionView::OnPageActionModelChanged(
   if (!model.GetVisible()) {
     ResetSlideAnimation(/*show=*/false);
     NotifyIsChipShowingChange();
-  } else if (model.ShouldShowSuggestionChip()) {
+  } else if (model.ShouldShowSuggestionChip() || model.GetAlwaysShowLabel()) {
     if (model.GetShouldAnimateChipIn()) {
       AnimateIn(/*string_id=*/std::nullopt);
     } else {
