diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 039c159cf496d1a97261cb77ee95516463086662..5832593526787cb08bb72b139bcea6a0f9b43aa8 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -567,6 +567,7 @@ void ToolbarView::Init() {
     media_button_ = AddChildView(std::move(media_button));
   }
 
+  BRAVE_TOOLBAR_VIEW_INIT
   avatar_ = AddChildView(std::make_unique<AvatarToolbarButton>(browser_view_));
   bool show_avatar_toolbar_button = true;
 #if BUILDFLAG(IS_CHROMEOS)
