diff --git a/chrome/browser/hub/internal/BUILD.gn b/chrome/browser/hub/internal/BUILD.gn
index 30ba0fe81da869256ce26d4d0f56ea8639df7035..e0e484370434d4428fe754c44b6d2e479cd43ff2 100644
--- a/chrome/browser/hub/internal/BUILD.gn
+++ b/chrome/browser/hub/internal/BUILD.gn
@@ -104,6 +104,7 @@ android_library("java") {
     "//ui/edge_to_edge:java",
     "//url:url_java",
   ]
+  sources += brave_browser_hub_internal_java_sources deps += brave_browser_hub_internal_java_deps
 }
 
 android_resources("java_resources") {
