diff --git a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h
index 239e34a7ca1bf7f1e6e74784fe561519038eada9..985d82d8dd0e15a890595755531f585f13f41eb2 100644
--- a/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h
+++ b/chrome/browser/ui/views/frame/horizontal_tab_strip_region_view.h
@@ -33,7 +33,7 @@ class TabStripControlButton;
 
 // Container for the tabstrip and the other views sharing space with it -
 // with the exception of the caption buttons.
-class HorizontalTabStripRegionView final : public TabStripRegionView {
+class HorizontalTabStripRegionView : public TabStripRegionView {
   METADATA_HEADER(HorizontalTabStripRegionView, TabStripRegionView)
 
  public:
