diff --git a/components/permissions/android/permission_prompt/permission_dialog_delegate.cc b/components/permissions/android/permission_prompt/permission_dialog_delegate.cc
index e570092babf63e94f9bc23ec8cb97886d113ec6f..eca9dcc5017f0d182f4441a56aa77b687dd79f43 100644
--- a/components/permissions/android/permission_prompt/permission_dialog_delegate.cc
+++ b/components/permissions/android/permission_prompt/permission_dialog_delegate.cc
@@ -59,6 +59,7 @@ void PermissionDialogJavaDelegate::CreateJavaDelegate(
 void PermissionDialogJavaDelegate::CreateDialog(
     content::WebContents* web_contents) {
   JNIEnv* env = base::android::AttachCurrentThread();
+  BRAVE_PERMISSION_DIALOG_PRE_CREATE_DIALOG
   // Send the Java delegate to the Java PermissionDialogController for display.
   // When the Java delegate is no longer needed it will in turn reset the native
   // java delegate (PermissionDialogJavaDelegate).
@@ -164,6 +165,7 @@ PermissionDialogDelegate::CreateForTesting(
 
 void PermissionDialogDelegate::Accept(JNIEnv* env) {
   CHECK(permission_prompt_);
+  BRAVE_PERMISSION_DIALOG_DELEGATE_ACCEPT
   permission_prompt_->Accept(prompt_options_);
 }
 
@@ -179,6 +181,7 @@ void PermissionDialogDelegate::Acknowledge(JNIEnv* env) {
 
 void PermissionDialogDelegate::Deny(JNIEnv* env) {
   CHECK(permission_prompt_);
+  BRAVE_PERMISSION_DIALOG_DELEGATE_CANCEL
   permission_prompt_->Deny(prompt_options_);
 }
 
