diff --git a/components/policy/resources/webui/policy_base.ts b/components/policy/resources/webui/policy_base.ts
index 2e576f866b6514f47847a6358ef2eb6d36945e0e..e52ab6813847dadb8d0649e30ba9e0f3b0e59071 100644
--- a/components/policy/resources/webui/policy_base.ts
+++ b/components/policy/resources/webui/policy_base.ts
@@ -213,7 +213,7 @@ export class Page {
                       policyNames['precedence']?.policyNames,
                     ].includes(knownPolicyNames) &&
                             knownPolicyNamesSet.has(name) ?
-                        `https://chromeenterprise.google/policies/?policy=${
+                        `https://support.brave.app/hc/en-us/articles/360039248271-Group-Policy?policy=${
                             name}` :
                         undefined,
                     isExtension: value.isExtension || false,
