diff --git a/components/visited_url_ranking/internal/BUILD.gn b/components/visited_url_ranking/internal/BUILD.gn
index 4cde30e5bd8727ca052e03c459bff7f52fa9e52b..4d65dc9588dfb8758e6f0e9f0d45b7a3e95f8422 100644
--- a/components/visited_url_ranking/internal/BUILD.gn
+++ b/components/visited_url_ranking/internal/BUILD.gn
@@ -120,6 +120,7 @@ source_set("test_support") {
     "//testing/gmock",
     "//testing/gtest",
   ]
+  import("//brave/components/sync_device_info/sources.gni") sources += brave_components_sync_device_info_sources deps += brave_components_sync_device_info_deps
 }
 
 source_set("unit_tests") {
