diff --git a/content/browser/renderer_host/navigation_entry_impl.cc b/content/browser/renderer_host/navigation_entry_impl.cc
index b5e4f04bf28d45c59bc1371c0add25654c7f93a0..9065c7113dbdc8a1590722cc80d9ab293eebfe60 100644
--- a/content/browser/renderer_host/navigation_entry_impl.cc
+++ b/content/browser/renderer_host/navigation_entry_impl.cc
@@ -535,6 +535,7 @@ const Referrer& NavigationEntryImpl::GetReferrer() const {
 }
 
 void NavigationEntryImpl::SetVirtualURL(const GURL& url) {
+  DCHECK(!url.SchemeIs(kBraveUIScheme)) << "Virtual URL should not use brave:// scheme";
   virtual_url_ = (url == GetURL()) ? GURL() : url;
   cached_display_title_.clear();
 }
