diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 6c20db955545b37cb1fd6fcc9b280aa1c7a14ae9..19e9acaf471f8821725fdb2951c53329659c8a89 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3754,6 +3754,7 @@ source_set("browser") {
       "surface_embed/surface_embed_connector_impl.h",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
