diff --git a/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.mm b/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.mm
index d21256326ab6b031545a032809c21fbe9251317c..55ef465710626602c6c24012979b2ad16b85c4a9 100644
--- a/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.mm
+++ b/content/app_shim_remote_cocoa/render_widget_host_view_cocoa.mm
@@ -2136,6 +2136,7 @@ static NSWindow* __weak _deferredResignKeyWindow;
 }
 
 - (id)accessibilityParent {
+  BRAVE_ACCESSIBILITY_PARENT
   if (_accessibilityParent)
     return NSAccessibilityUnignoredAncestor(_accessibilityParent);
   return [super accessibilityParent];
