diff --git a/services/network/public/cpp/permissions_policy/permissions_policy_features.json5 b/services/network/public/cpp/permissions_policy/permissions_policy_features.json5
index c01ac85d8c49a05783321a9e39227178ad2eb478..1c9b85c30eee85f48aaac7b0095a9d186387bfa8 100644
--- a/services/network/public/cpp/permissions_policy/permissions_policy_features.json5
+++ b/services/network/public/cpp/permissions_policy/permissions_policy_features.json5
@@ -668,6 +668,18 @@
       permissions_policy_name: "writer",
       depends_on: ["AIWriterAPI"],
     },
+    {
+      name: "Ethereum",
+      permissions_policy_name: "ethereum"
+    },
+    {
+      name: "Solana",
+      permissions_policy_name: "solana"
+    },
+    {
+      name: "Cardano",
+      permissions_policy_name: "cardano"
+    },
   ],
   // LINT.ThenChange(//third_party/blink/public/devtools_protocol/domains/Page.pdl:PermissionsPolicyFeature)
 }
