diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
index 4250447f24427ccd9cf0efa589d40eaf0a7cb701..1f67568a9b338341b3c0e53702d5b842d7888b5f 100644
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -869,6 +869,11 @@
       public: true,
       status: "stable",
     },
+    {
+      name: "BraveIsInTorContext",
+      public: true,
+      base_feature: "none",
+    },
     {
       name: "BrowserInitiatedAutomaticPictureInPicture",
       public: true,
@@ -2594,6 +2599,7 @@
       // In-development features for the File System Access API.
       name: "FileSystemAccessAPIExperimental",
       status: "experimental",
+      base_feature: "none",
     },
     {
       // The FileSystemHandle.getCloudIdentifiers() method (see
@@ -2608,6 +2614,7 @@
       // Non-OPFS File System Access API.
       name: "FileSystemAccessLocal",
       status: {"default": "stable"},
+      base_feature: "none",
     },
     {
       name: "FileSystemAccessLockingScheme",
@@ -3699,6 +3706,7 @@
     {
       name: "MiddleClickAutoscroll",
       status: "test",
+      base_feature: "none",
     },
     // Killswitch for crbug.com/40062462 fix.
     {
