diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index d1707fd3d80fa31327984edd040c63687ff237d2..1cd7e217cddc0bc6cc83883cb4d3c5de54cc26fe 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2755,6 +2755,7 @@ grd_files_unbundled_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_unbundled_sources += brave_grd_files_unbundled_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_bundled_sources =
